components/eventing-controller/controllers/subscription/nats/unit_test.go

```diff
@@ -196,94 +200,93 @@ func Test_syncSubscriptionStatus(t *testing.T) {
 
 	testCases := []struct {
 		name                  string
 		givenSubscriptionSink string
 		givenSvcNameToCreate  string
-		expectedErrString     string
+		wantErrString         string
 	}{
 		{
 			name:                  "With invalid scheme",
 			givenSubscriptionSink: "invalid Sink",
-			expectedErrString:     "sink URL scheme should be 'http' or 'https'",
+			wantErrString:         "sink URL scheme should be 'http' or 'https'",
 		},
 		{
 			name:                  "With invalid URL",
 			givenSubscriptionSink: "http://invalid Sink",
-			expectedErrString:     "not able to parse sink url with error",
+			wantErrString:         "not able to parse sink url with error",
 		},
 		{
 			name:                  "With invalid suffix",
 			givenSubscriptionSink: "https://svc2.test.local",
-			expectedErrString:     "sink does not contain suffix",
+			wantErrString:         "sink does not contain suffix",
 		},
 		{
 			name:                  "With invalid suffix and port",
 			givenSubscriptionSink: "https://svc2.test.local:8080",
-			expectedErrString:     "sink does not contain suffix",
+			wantErrString:         "sink does not contain suffix",
 		},
 		{
 			name:                  "With invalid number of subdomains",
 			givenSubscriptionSink: "https://svc.cluster.local:8080", // right suffix but 3 subdomains
-			expectedErrString:     "sink should contain 5 sub-domains",
+			wantErrString:         "sink should contain 5 sub-domains",
 		},
 		{
 			name:                  "With different namespaces in subscription and sink name",
 			givenSubscriptionSink: "https://eventing-nats.kyma-system.svc.cluster.local:8080", // sub is in test ns
-			expectedErrString:     "namespace of subscription: test and the namespace of subscriber: kyma-system are different",
+			wantErrString:         "namespace of subscription: test and the namespace of subscriber: kyma-system are different",
 		},
 		{
 			name:                  "With no existing svc in the cluster",
 			givenSubscriptionSink: "https://eventing-nats.test.svc.cluster.local:8080",
-			expectedErrString:     "sink is not valid cluster local svc, failed with error",
+			wantErrString:         "sink is not valid cluster local svc, failed with error",
 		},
 		{
 			name:                  "With no existing svc in the cluster, service has the wrong name",
 			givenSubscriptionSink: "https://eventing-nats.test.svc.cluster.local:8080",
 			givenSvcNameToCreate:  "test", // wrong name
-			expectedErrString:     "sink is not valid cluster local svc, failed with error",
+			wantErrString:         "sink is not valid cluster local svc, failed with error",
 		},
 		{
 			name:                  "With no existing svc in the cluster, service has the wrong name",
 			givenSubscriptionSink: "https://eventing-nats.test.svc.cluster.local:8080",
 			givenSvcNameToCreate:  "eventing-nats",
-			expectedErrString:     "",
+			wantErrString:         "",
 		},
 	}
 	for _, tC := range testCases {
 		testCase := tC
```

Comment:
I think this is an additional assignment that is not needed. You can write `testCase` instead of `tC` in the above line. Same comment for all occurrences in this file.

Reply:
https://stackoverflow.com/questions/68559574/using-the-variable-on-range-scope-x-in-function-literal-scopelint

Reply:
At the point of time it was written, the IDE was complaining about it, strangely not anymore, I will remove it

Reply:
Yes, the `golangci-lint` is complaining about ` Using the variable on range scope `testCase` in function literal` in that case